app/views/shared/_msg.html.erb

```diff
@@ -0,0 +1,10 @@
+<% if notice  %>
+<div class="alert alert-warning" role="alert">
+  <%= notice %>
+</div>
+<% end  %>
+<% if alert  %>
+<div class="alert alert-danger" role="alert">
+  <%= alert %>
+</div>
+<% end  %>
```

Comment:
- [ ]  We don't need to add any logic inside the .erb file, we use the helper folders to create methods with logic and called inside the .erb file. You can refer to this comment to modify the files that have logic inside the .erb file.

Read [this](https://www.toptal.com/ruby-on-rails/top-10-mistakes-that-rails-programmers-make) for more information